k6/src/k6/k6-configuration.ts

```
@@ -0,0 +1,154 @@
+        }
+    }
+
+    private loadEnvironmentVariables(): void {
+        this.clusterNodes = __ENV.K6_CLUSTER_NODES || "core";
+        this.topology = __ENV.K6_TOPOLOGY_NAME || "many2many";
+        this.workload = __ENV.K6_WORKLOAD_NAME || "sanity-check";
+
+        __ENV.K6_WEBSOCKET_DISCONNECTED = "false";
+        if (__ENV.REQUESTS_PER_SECOND_PER_VU) {
+            const rps = parseInt(__ENV.REQUESTS_PER_SECOND_PER_VU);
+            if (!isNaN(rps) && rps > 0) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)